### PR TITLE
LOG-13599 Attempt to fix Travis builds

### PR DESCRIPTION
**Pull request by rjacobs-r7**

### CODE REVIEW

**Comment by rjacobs-r7** [comment]
@mzabad-r7 `log.error("failure", new RuntimeException("error").initCause(new RuntimeException("cause")));` -- Expected :)

File: pom.xml [hunk 1]
```diff
@@ -43,8 +43,8 @@
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>2.0.2</version>
                 <configuration>
-                    <source>1.6</source>
-                    <target>1.6</target>
+                    <source>1.8</source>
```
Comment (jcolgan-r7) [inline pom.xml]: Won't this break compatibility with <8? If we're not using any features from 8+, I don't see the need

Comment (rjacobs-r7) [reply pom.xml]: Even 8 is EOL, there's no point in supporting Java 6 and 7 imo. People who need Java 6 support can stay on le_java for the time being. The tests for them currently don't work either and I don't see a point in fixing tests for dead Java versions.

Comment (rjacobs-r7) [reply pom.xml]: Also Java 12 will actually not even run Java 6 bytecode anymore, as it turns out.

**Pull request merged**